code/lib/manager-api/src/tests/refs.test.ts

```diff
@@ -171,6 +171,9 @@ describe('Refs API', () => {
       // given
       initRefs({ provider, store } as any);
 
+      // the `runCheck` is async, so we need to wait for it to finish
+      await Promise.resolve();
```

Comment:
I'm open to other ideas on how to ensure the unawaited promise is resolved.. 

Maybe I should try this:
https://vitest.dev/api/vi.html#vi-waitfor